### DISCUSSION TITLE
TreeView 选中节点状态不正确

### DISCUSSION THREAD

**Discussion by lilililililili-li** [dotnetcore/BootstrapBlazor#3505] [Q&A] [answered]

**Comment by bb-auto** [comment]
@henry-ll Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment]
@henry-ll 如图所示，我感觉是对的啊。确实有一个节点没有选中，父级节点状态就是未确定状态。

你的问题是：**你期望父级节点状态是选中吗？**

**Comment by lilililililili-li** [comment]


**Reply by ArgoZhang** [reply]
懂你意思啦。我稍后复现，修复一下

**Reply by ArgoZhang** [reply]
@henry-ll 点击撤销结果点击成了移除本群，误操作。不过还是要感谢你提交组件的问题，是问题我们就会修复的

**Reply by ArgoZhang** [reply, marked as answer]
link #3517
